index.html

```diff
@@ -950,11 +956,257 @@ <h3>
+          information lies with the user.
+          </li>
+        </ul>
+        <p>
+          For a more detailed exploration of how to determine and address
+          unnecessary usage, it makes sense to consider goverment-issued
+          credentials and other credentials separately, as they potentially
+          differ in the sensitivity of their data and the potential harms from
+          misuse as well as legal & regulatory considerations.
+        </p>
+        <p>
+          A key component of risk mitigation and ensuring user control that
+          applies to both types of credentials is the user agent's ability to
+          inspect the credential request metadata and make decisions or UI
+          presentation based on it. The DC API ensures this user agent access
+          to the exchange through requirements placed on protocols to transmit
+          requests in plain text and to include relevant information.
+        </p>
+        <h4>
+          Government-issued credentials
+        </h4>
+        <p>
+          <a data-cite=
+          "identity-web-impact#pure-digital-credentials">Government-issued
+          digital credentials</a> include travel documents, personal licenses,
+          proof of welfare and public health programs, vehicle registrations
+          and other documents issued by government authorities, or other
+          documents representing this information. These documents are highly
+          sensitive, as they contain permanent, irrevocable, unique identifiers
+          that are central to a person's individual identity and ability to
+          interact with vital public services.
+        </p>
+        <h5>
+          Risk of theft and leakage of government credentials
+        </h5>
+        <p>
+          The high value of these credentials to users and attackers means
+          there is a significant risk of theft, and significant potential harm
+          from leakage to unauthorized third parties. This includes the request
+          of government identity for the purpose of tracking and
+          personalization.
+        </p>
+        <h5>
+          Risk of proliferation of requests for government credentials
+        </h5>
+        <p>
+          A major concern with increased availability of government credentials
+          online is <a href=
+          "https://en.wikipedia.org/wiki/Jevons_paradox">Jevon's Paradox</a>,
+          i.e., the chance of increasing demand for credentials through lower
+          friction of access. This might result in harmful outcomes for users:
+        </p>
+        <ul>
+          <li>Increased risk of information leakage, and ultimately a less
+          trusted user experience on the Web. When a large number of services
+          access and store government-issued credentials, the chance of data
+          leaks and unauthorized access increases as well. Even seemingly
```

Comment:
Maybe... but the response are encrypted by design... but yes, a developer could be silly and store the decrypted credentials on a server (which would be self defeating, but humans do silly things).

Reply:
Put differently, the point of ISO18013 and OpenID4VP is that the developers don't need to store the unencrypted version (unlike today, where people take a photo of their credential) - so the point is to reduce fraud, identity theft, etc. If the encrypted data is stolen, it's useless unless the attacker also steals the private keys. They can't decrypt the stolen credentials that are on the server without the keys, so the data is useless.

Reply:
Yeah, that's a valid point... I should probably work that in, I can take a look and otherwise file a follow-up issue.